include/LadspaControl.h

```diff
@@ -35,8 +35,7 @@
 namespace lmms
 {
 
-
-typedef struct PortDescription port_desc_t;
+using port_desc_t = struct PortDescription;
```

Comment:
Basing off of my previous point in https://github.com/LMMS/lmms/pull/6453#discussion_r910516604, I think it will be easier if we just declare the type using ``struct port_desc_t``. 

And thanks to the rationale from the C++ standard:
```
For ease of use, C++ doesn’t require that a type name be prefixed with the keywords class, struct or union when used in object declarations or type casts.
```

This shouldn't affect the surrounding code.

Reply:
I went ahead and just included ``LadspaBase.h`` and turned ``using port_desc_t = struct PortDescription;`` to just ``struct port_desc_t`` in LadspaBase.h.

Reply:
Oh no, that's not good. Including files include compile time. There were large efforts to remove useless includes (#5860). As PhysSong explained, this is a forward declaration and the `struct` has to be used. So, can you please revert this change of including new files?